.solcover.js

```diff
@@ -7,8 +20,13 @@ module.exports = {
       'testHelpers/NoLimitSubdomains',
       'testHelpers/TaskSkillEditing'
     ],
-    copyPackages: ['openzeppelin-solidity'],
-    compileCommand: 'yarn run provision:token:contracts',
-    testCommand: '../node_modules/.bin/truffle test --network coverage',
-    testrpcOptions: `--port 8555 -i 1999 --acctKeys="./coverageEnv/ganache-accounts.json" --noVMErrorsOnRPCResponse --accounts 12 --allowUnlimitedContractSize`
-};
+    providerOptions: {
```

Comment:
These are all the flags passed to testrc-sc, translated to ganache-core options and passed to ganache.